.github/workflows/release-secure-signal-examples-docker-image-standard.yaml

```diff
@@ -0,0 +1,34 @@
+name: Release Secure Signal Examples Docker Image for Standard
+run-name: ${{ inputs.release_type == 'Snapshot' && 'Publish Pre-release' || format('Release {0}', inputs.release_type)}} Secure Signal Examples Docker Image for Standard by @${{ github.actor }}  
+on:
+  workflow_dispatch:
+    inputs:
+      release_type:
+        type: choice
+        description: 'The type of release'
```

Comment:
Same comment here about quotes and `required: true`

Reply:
Because it's a `type: choice`, I don't think we need to add `required: true` as it's a dropdown menu